src/Disks/ObjectStorages/CommonPathPrefixKeyGenerator.cpp

```diff
@@ -51,15 +51,14 @@ std::tuple<std::string, std::vector<std::string>> CommonPathPrefixKeyGenerator::
     std::deque<std::string> dq;
```

Comment:
These are variable names I can't stand.

Reply:
In a separate PR, the hierarchical key generator should be removed as the 24.6 was the last version where it was used, and the disk was not in the cloud at that time.
A heuristic to select either hierarchical or flat layout is not ideal, either.
It will be a backward incompatible change, however.

Reply:
Yes, I wanted to ask about this. Let's remove it :) It's unlikely to have production usage of the old layout anywhere.